pkg/stanza/operator/input/windows/input.go

```diff
@@ -112,6 +196,17 @@ func (i *Input) readToEnd(ctx context.Context) {
 			return
 		default:
 			if count := i.read(ctx); count == 0 {
+				if i.isRemote() && i.remoteSessionHandle == 0 {
+					if i.remoteSessionHandle == 0 {
```

Comment:
This condition will always be true because it's required in the previous line.

Reply:
True. Updated.